flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/common/CommonExecPythonCorrelate.java

```diff
@@ -88,7 +88,7 @@ public CommonExecPythonCorrelate(
         final Transformation<RowData> inputTransform =
                 (Transformation<RowData>) inputEdge.translateToPlan(planner);
         final Configuration pythonConfig =
-                CommonPythonUtil.getMergedConfig(planner.getExecEnv(), config.getTableConfig());
+                CommonPythonUtil.getMergedConfig(planner.getExecEnv(), config.getPlannerConfig());
```

Comment:
As mentioned before, I would strongly vote for still calling this `TableConfig`.